clang/test/CodeGen/aarch64-sme2-intrinsics/acle_sme2_max.c

```diff
@@ -0,0 +1,1562 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+// RUN: %clang_cc1 -D__ARM_FEATURE_SME -D__ARM_FEATURE_SME2 -triple aarch64-none-linux-gnu -target-feature +sve -target-feature +sme2 -S -disable-O0-optnone -Werror -Wall -emit-llvm -o - %s | opt -S -p mem2reg,instcombine,tailcallelim | FileCheck %s
```

Comment:
I think the `-D__ARM_FEATURE` flags can be removed as you're also passing `-target-feature`